client/material/left-nav/clickable-subheader.tsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react'
+import styled from 'styled-components'
+import { Link, useRoute } from 'wouter'
+import { colorTextPrimary, colorTextSecondary } from '../../styles/colors'
+import { overline, singleLine } from '../../styles/typography'
+
+const Container = styled.div<{ $isActive: boolean }>`
+  width: 100%;
+  height: 36px;
+  background-color: ${props => (props.$isActive ? 'rgba(255, 255, 255, 0.08)' : 'transparent')};
+  color: ${props => (props.$isActive ? colorTextPrimary : colorTextSecondary)};
+  cursor: pointer;
+
+  &:hover {
```

Comment:
As stated below, please just handle this type of stuff through a ripple instead of adding a bespoke hover state